class.jetpack.php

```diff
@@ -4526,7 +4531,9 @@ function build_connect_url( $raw = false, $redirect = false, $from = false, $reg
 			 */
 			$auth_type = apply_filters( 'jetpack_auth_type', 'calypso' );
 
-			$tracks_identity = jetpack_tracks_get_identity( get_current_user_id() );
+
+			$tracks = new Automattic\Jetpack\Tracking();
```

Comment:
We don't need the namespace here, do we?

```suggestion
			$tracks = new Tracking();
```